collectd_transmission/__init__.py

```diff
@@ -99,9 +102,9 @@ def field_getter(stats, key, category):
         int. The metric value or 0
     '''
     if category == 'cumulative':
-        return stats.cumulative_stats[key]
+        return getattr(stats.cumulative_stats, key)
```

Comment:
This is slightly less performant as an implementation. I could see it used if there was a default returned (for which I don't have a good value) but without a default, I am wondering what is the incentive behind the change?

Reply:
Disregard, I figured out why you needed to change that, it's cause a SessionStats instance can't be indexed by key and statistics are attributes instead.